pkg/apis/core/validation/validation.go

```diff
@@ -1524,6 +1524,7 @@ var supportedVolumeModes = sets.NewString(string(core.PersistentVolumeBlock), st
 
 var supportedDataSourceAPIGroupKinds = map[schema.GroupKind]bool{
 	{Group: "snapshot.storage.k8s.io", Kind: "VolumeSnapshot"}: true,
+	{Group: "", Kind: "PersistentVolumeClaim"}:                 true,
```

Comment:
I think core objects should still have a group

Reply:
@msau42 It seemed redundant to require it based on the definition here:  https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L4062-L4065

The object checks default to the core type so it's not needed for any of the implementation.

Reply:
Core API's group is "".  That is why we had to make APIGroup optional.